package.json

```diff
@@ -21,8 +22,8 @@
     "build:js": "babel src --out-dir lib --extensions \".ts\" --source-maps inline --verbose",
     "prebuild": "node utils/update-puppeteer-config.js",
     "postbuild": "rimraf lib/tests",
-    "prepare:core": "node utils/prepare-israeli-bank-scrapers-core.js && npm i --package-lock-only && npm ci && npm run lint -- --fix && npm run build",
-    "prepare:default": "npm ci && npm run build",
+    "prepare:core": "git reset --hard && node utils/prepare-israeli-bank-scrapers-core.js && npm i --package-lock-only && npm ci && npm run lint -- --fix && npm run build",
+    "prepare:default": "git reset --hard && npm ci && npm run build",
     "reset": "git reset --hard && npm ci",
     "publish:core": "npm run prepare:core && npm publish --access public && npm run reset",
     "publish:default": "npm run prepare:default && npm test && npm publish --access public && npm run reset",
```

Comment:
these shouldn't work anymore, right?
I mean, if we want to keep them for manual publish, perhaps we should fix them as well?

Reply:
removed